### PR TITLE
jobs: update page

### PR DESCRIPTION
**Pull request by 4G3NT**
Similar to my last pull request: #10301

I made changes to clarify that the `jobs` command is not exclusive to BASH. Additionally, I added distinguishing markers for the options that are only to BASH.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
jobs

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/linux/jobs.md [hunk 1]
```diff
@@ -11,18 +12,18 @@
...
 
 `jobs -p`
 
-- Display running processes:
+- Display running processes (~):
 
 `jobs -r`
 
-- Display stopped processes:
+- Display stopped processes (~):
```
Comment (acuteenvy) [inline pages/linux/jobs.md:27]:
```suggestion
- Display stopped processes:
```

File: pages/linux/jobs.md [hunk 2]
```diff
@@ -11,18 +12,18 @@
...
+- Display information about jobs with changed status (~):
 
 `jobs -n`
 
 - Display process ID of process group leader:
 
 `jobs -p`
 
-- Display running processes:
+- Display running processes (~):
```
Comment (acuteenvy) [inline pages/linux/jobs.md:23]:
```suggestion
- Display running processes:
```

File: pages/linux/jobs.md [hunk 3]
```diff
@@ -11,18 +12,18 @@
 
 `jobs -l`
 
-- Display information about jobs with changed status:
+- Display information about jobs with changed status (~):
 
 `jobs -n`
 
 - Display process ID of process group leader:
```
Comment (acuteenvy) [inline pages/linux/jobs.md:19]:
```suggestion
- Display only process IDs:
```

**Review by acuteenvy — COMMENTED** [review]
I find the `~` notation a little weird. It's not used anywhere else, so I think writing what options are exclusive to `bash` in the description is better.

**Review by 4G3NT — COMMENTED** [review]
👍

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**